run_sample_httpserver.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 # Script to run a java application for testing jmx4prometheus.
-dctry=$(dirname $0)
-builddir=${dctry}/target/
 
-java -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.port=5555 -cp ${builddir}/jmx4prometheus-0.2-tests.jar com.prometheus.jmx4prometheus.Main
+java -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.port=5555 -jar httpserver/target/httpserver-0.3-SNAPSHOT-jar-with-dependencies.jar 5556 sample_config.json
```

Comment:
I notice the exposed port is on the commandline but the target port is in the config. Any reason for the two ways of configuring?